src/test/app/MultiSign_test.cpp

```diff
@@ -1672,6 +1672,29 @@ class MultiSign_test : public beast::unit_test::suite
         BEAST_EXPECT(env.seq(alice) == aliceSeq + 1);
     }
 
+    void
+    test_SignerListSet_Flags(FeatureBitset features)
```

Comment:
Please use proper `camelCase` (I know it doesn't match the other tests in these files, but I'd rather stick to proper norms)

Reply:
renamed